spec/requests/api/v0.1/post_hesa_trainees_spec.rb

```diff
@@ -136,6 +135,13 @@
       expect(response.parsed_body["errors"]).to include("Course subject one can't be blank")
       expect(response.parsed_body["errors"]).to include("Study mode can't be blank")
       expect(response.parsed_body["errors"]).to include("Email Enter an email address in the correct format, like name@example.com")
+      expect(response.parsed_body["errors"]).to include("Itt aim can't be blank")
+      expect(response.parsed_body["errors"]).to include("Itt qualification aim can't be blank")
+      expect(response.parsed_body["errors"]).to include("Course year can't be blank")
+      expect(response.parsed_body["errors"]).to include("Course age range can't be blank")
+      expect(response.parsed_body["errors"]).to include("Fund code can't be blank")
+      expect(response.parsed_body["errors"]).to include("Funding method can't be blank")
+      expect(response.parsed_body["errors"]).to include("Hesa can't be blank")
```

Comment:
Should that be 
`Hesa id can't be blank`?
```suggestion
      expect(response.parsed_body["errors"]).to include("Hesa id can't be blank")
```

Reply:
rails does it "thing" and it's humanising the error message - it drops the "id" from any attribute names